jenkins/jenkins.sh

```diff
@@ -236,10 +236,10 @@ then
     tar czf "${VG_VERSION}_output.tar.gz" vgci-work test-report.xml jenkins/vgci.py jenkins/jenkins.sh vgci_cfg.tsv
     aws s3 cp --acl public-read "${VG_VERSION}_output.tar.gz" s3://cgl-pipeline-inputs/vg_cgl/vg_ci/jenkins_output_archives/
 
-    # if success and we're merging the PR (and not just testing it), we publish results to the baseline
-    if [ "$PYRET" -eq 0 ] && [ -z ${ghprbActualCommit} ]
+    # if we're merging the PR (and not just testing it), we publish results to the baseline
+    [ -z ${ghprbActualCommit} ]
```

Comment:
@glennhickey Looks like you dropped the if; I think baseline might be getting updated from PRs now? Or maybe never updated?

Reply:
Also looks like you fixed it yesterday.